apps/ledger-live-mobile/src/components/Nft/NftViewer.tsx

```diff
@@ -291,142 +294,162 @@ const NftViewer = ({ route }: Props) => {
     [nftMetadata, nftStatus],
   );
 
+  const [isOpen, setOpen] = useState<boolean>(false);
+  const onCloseModal = useCallback(() => {
+    setOpen(false);
+  }, []);
+  const onOpenModal = useCallback(() => {
+    setOpen(true);
+  }, []);
+  const isNFTDisabled = useFeature("disableNftSend")?.enabled;
+
   return (
-    <View>
-      <ScrollView contentContainerStyle={styles.scrollView}>
-        <View style={styles.main}>
-          <Skeleton
-            style={[styles.tokenName, styles.tokenNameSkeleton]}
-            loading={isLoading}
-          >
-            <LText style={styles.tokenName}>
-              {collectionMetadata?.tokenName || "-"}
-            </LText>
-          </Skeleton>
+    <>
+      <InfoModal
+        isOpened={!!isOpen}
```

Comment:
not important but since `isOpen` is a `boolean` that shouldn't be necessary, right ?